ui/jest.config.js

```diff
@@ -1,9 +1,13 @@
 module.exports = {
   preset: "ts-jest",
+  testEnvironment: "jsdom",
+  setupFilesAfterEnv: ["<rootDir>/setupTests.ts"],
   transformIgnorePatterns: [
     "node_modules/(?!(argo-ui)/)"
   ],
   moduleNameMapper: {
-    "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js"
+    "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js",
+    "\\.(jpg|jpeg|png|gif|webp|svg)$": "<rootDir>/__mocks__/fileMock.js"
   },
+  moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json", "node"],
```

Comment:
Not sure why this was changed, but this is a subset of the default for this setting, so I think it's better to stick with the default:
```suggestion
```